src/iOS/toga_iOS/libs/uikit.py

```diff
@@ -326,6 +326,49 @@ class UIProgressViewStyle(Enum):
 # UISwitch.h
 UISwitch = ObjCClass('UISwitch')
 
+######################################################################
+# UITabBar.h
+
+UITabBar = ObjCClass('UITabBar')
+
+UITabBarItemPositioningAutomatic = 0
```

Comment:
This hasn't been backported everywhere,but we've been trying to use Enums for constants like this that are clearly grouped - see [NSAlertType in the cocoa backend](https://github.com/beeware/toga/blob/master/src/cocoa/toga_cocoa/libs/appkit.py#L27) for an example.

Reply:
Awesome.  I much prefer enums, but I fought really hard to keep the existing style.  Happy to migrate (at least the bits relevant for this PR).